finalPrep/annotations.js

```diff
@@ -2,41 +2,94 @@
 // Use the double // to create a new comment
```

Comment:
BEAUTIFUL annotations - everything I see is spot on. I appreciated your level of precision and use of technical vocabulary.